sda-commons-server-mongo-testing/src/test/java/org/sdase/commons/server/mongo/testing/MongoDbVersionTest.java

```diff
@@ -0,0 +1,63 @@
+package org.sdase.commons.server.mongo.testing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import de.flapdoodle.embed.mongo.distribution.IFeatureAwareVersion;
+import de.flapdoodle.embed.mongo.distribution.Version;
+import java.util.concurrent.atomic.AtomicReference;
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.Test;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+public class MongoDbVersionTest {
```

Comment:
Do we want new tests to be JUnit 5?

Reply:
They are not really "new" I just moved them to a new class. I would keep them in Junit4